Library/Homebrew/tap.rb

```diff
@@ -1401,6 +1401,13 @@ def new_cask_path(token)
     cask_dir/cask_subdir/"#{token.downcase}.rb"
   end
 
+  sig { params(token: String).returns(Pathname) }
+  def new_font_path(token)
+    font_first_letter = T.must(token.split("-").second)[0].to_s
+    cask_subdir = "fonts/font-#{font_first_letter}"
```

Comment:
Have you (or anyone) done any testing to ensure the doubly nested subdirectories approach works as expected? This would be the first time we've tried it.

Reply:
> `brew info` worked, but was not quite right so that needs an update as it did not report the nested directories. Not sure if that's because the path doesn't actually exist, or some other update needs done here. So I'm happy to look but knowing this if you want to merge.

Yeh, that would be good to fix before we migrate the rest. I suspect there may be some other edge-cases.